project/MiMaFilters.scala

```diff
@@ -14,12 +15,16 @@ object MiMaFilters {
     ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.LazyVals.Evaluating"),
     ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.LazyVals.NullValue"),
 
+    //  New experimental features in 3.3.X
+    ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language#experimental.clauseInterleaving"),
+    ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$experimental$clauseInterleaving$"),
     ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language#experimental.into"),
     ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$experimental$into$"),
     ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language#experimental.pureFunctions"),
     ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language#experimental.captureChecking"),
     ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$experimental$pureFunctions$"),
     ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$experimental$captureChecking$"),
+    // /New experimental features in 3.3.X
```

Comment:
```suggestion
    // New experimental features in 3.3.X
```
But perhaps it's already redundant with the title above?

Reply:
```suggestion
    // End of new experimental features in 3.3.X
```

Is this better ?
The idea is to clearly mark both the beginning and end of the experimental features
The entries below this comment are no longer experimental features

Reply:
Oh. Er, yes, definitely. That is not at all how I had interpreted the `/`.